app/views/static/test.html

```diff
@@ -1,2 +0,0 @@
-<!-- Please do not remove, will break tests :) -->
-It works!
```

Comment:
@abonas this is OK, as long as `spec/requests/static_spec.rb` is fixed to check some other file, it was a placeholder for when we have something real to test. Though, @zeari, I don't see any static_spec changes here yet..